force-app/emails/labels/CustomLabels.labels

```diff
@@ -0,0 +1,275 @@
+    <labels>
+        <fullName>emailErrorGetFromAddress</fullName>
+        <categories>Email,Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when fail to retrieve from address options</shortDescription>
+        <value>Unable to retrieve originator options.</value>
+    </labels>
+    <labels>
+        <fullName>emailErrorGetRecipients</fullName>
+        <categories>Email,Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when unable to get recipient info</shortDescription>
+        <value>Unable to retrieve recipient information.</value>
+    </labels>
+    <labels>
+        <fullName>emailErrorInputRequired</fullName>
+        <categories>Email,Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when required field is empty</shortDescription>
+        <value>Complete this field.</value>
+    </labels>
+    <labels>
+        <fullName>emailErrorMessageLabel</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for column in delivery report</shortDescription>
+        <value>Error Message</value>
+    </labels>
+    <labels>
+        <fullName>emailFromLabel</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for From field on email form</shortDescription>
+        <value>From</value>
+    </labels>
+    <labels>
+        <fullName>emailMessageLabel</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for Message field on email form</shortDescription>
+        <value>Message</value>
+    </labels>
+    <labels>
+        <fullName>emailNotSentHeading</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Heading for delivery report</shortDescription>
+        <value>The following emails could not be sent.</value>
+    </labels>
+    <labels>
+        <fullName>emailRecipientCountLabel</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label shown when too many recipients</shortDescription>
+        <value>{0} Recipients</value>
+    </labels>
+    <labels>
+        <fullName>emailRecipientNameLabel</fullName>
+        <categories>Email</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Column heading for recipients</shortDescription>
+        <value>Recipient Name</value>
+    </labels>
+    <labels>
+        <fullName>emailRecipientsInvalid</fullName>
```

Comment:
I didn't see this when I emailed a list that included people who were missing email addresses. Was this supposed to be included in a warning before send? I just completed the send flow and saw this after sending. 

![Screenshot 2024-08-16 at 10 24 57 AM](https://github.com/user-attachments/assets/bd59d554-1899-403e-9754-549e5f8335fa)

Reply:
After investigating this, I think what might have been happening in your testing is that if _all_ recipients do not have an email address then it interprets this as the user not having FLS access to the field because the existing logic included the faulty assumption that if all emails were missing then it must have been because of a lack of access, not because the data simply didn't have those values populated.

I updated this logic with an explicit check for FLS access for the Contact.Email field, and now hopefully you would not see the issue again in your testing.  Note that the warning of blank email addresses prior to sending does not show if the user does not have access to the Contact.Email field, so that is still a prerequisite, but now FLS access is more reliably determined.

Review commit 82d8c44 for details on what was incorrect before and what has been changed to fix the faulty logic.